tests/sentry/integrations/slack/notifications/test_assigned.py

```diff
@@ -142,5 +142,5 @@ def test_assignment(self, mock_func):
         assert attachment["title"] == self.group.title
         assert (
             attachment["footer"]
-            == f"{self.project.slug} | <http://testserver/settings/account/notifications/workflow/?referrer=assigned-activity-slack-user|Notification Settings>"
+            == f"{self.project.slug} | <http://testserver/settings/account/notifications/workflow/?referrer=assigned_activity-slack-user|Notification Settings>"
```

Comment:
@leeandher The referrer is part of the spec because we use it for analytics. If you change the referrer, a lot of charts will break or require changes.